anadi/__main__.py

```diff
@@ -19,7 +20,7 @@ def run():
         app.init(args.dir, os.path.abspath(os.path.expanduser(ANADI_CONF_FILE)))
         if args.query:
             csvdb = CSVDB()
-            csvdb.load(args.dir,SettingsDB())
+            csvdb.load(args.file,SettingsDB())
             result = csvdb.raw_sql(args.query)
             print(result)
```

Comment:
if *-d* has no value we need to chcek if both *-q* and *-f* have value.

Reply:
Now either the dir or the file and query need to be passed. Hence this as well taken care